app/view_models/auction_list_item.rb

```diff
@@ -0,0 +1,193 @@
+class AuctionListItem
+  attr_reader :auction, :current_user
+
+  def initialize(auction:, current_user:)
+    @auction = auction
+    @current_user = current_user
+  end
+
+  def title
+    auction.title
+  end
+
+  def id
+    auction.id
+  end
+
+  def html_summary
+    return '' if auction.summary.blank?
+    markdown.render(auction.summary).html_safe
+  end
+
+  def label
+    status_presenter.label
+  end
+
+  def label_class
+    status_presenter.label_class
+  end
+
+  def capitalized_formatted_type
+    if auction.type == "single_bid"
+      'single-bid'
+    else
+      'multi-bid'
+    end.capitalize
+  end
+
+  def eligibility_label
+    if for_small_business?
+      'Small-business only'
+    else
+      'SAM.gov only'
+    end
+  end
+
+  def available?
+    auction_status.available?
+  end
+
+  def current_user_can_bid?
+    if current_user.is_a?(Guest)
+      true
+    elsif !current_user.sam_accepted?
+      false
+    elsif for_small_business? && current_user.small_business?
+      true
+    elsif for_small_business? && !current_user.small_business?
+      false
+    else # not for small business
+      true
+    end
+  end
+
+  def bid_button_partial
+    if current_user_can_bid?
+      'auctions/bid_button'
+    else
+      'components/null'
```

Comment:
yeah we are trying to remove all conditionals from the view. 

The benefit of doing that is that it allows us to see in the ViewModel what the branches are. Out of those branches we will see patterns and be able to extract some more classes. 

If the conditionals are hidden in the views, it is harder to see patterns and therefore harder to extract objects

We aren't to the object extraction phase yet, but we're gonna get there!